docs/dotnet-five.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What is .NET 5
+description: In this article you will learn about .NET 5, a cross platform, and open-source project that convergences .NET framework and .NET Core together.
+ms.date: 08/31/2020
+ms.topic: overview
+ms.author: dapine
+author: IEvangelist
+---
+
+# What is .NET 5
+
+The .NET ecosystem is being simplified with the convergence of .NET Core and .NET Framework. Naming things is hard, but unifying versions of two disparate .NET implementations can be harder. This article details .NET 5, which can be thought of as .NET Core vNext. With .NET Core concluding at version 3.1 and .NET Framework at version 4.8, to avoid ambiguity the next logical version is .NET version 5.0.
+
+## Evolution of .NET
```

Comment:
I would really bring the point here by making this title, ".NET 5 is the next evolution of .NET Core" or something to that effect. As per  #19979 we want to make it very clear that it will _not_ bring along .NET Framework technologies like webforms or wcf

Reply:
Basically I would want readers to already come to that conclusion before any specific section talking about what it doesn't bring over

Reply:
Awesome, that makes a lot of sense. I'll incorporate that into this article.